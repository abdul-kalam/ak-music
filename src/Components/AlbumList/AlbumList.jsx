import React from 'react';
import { AlbumListWrapper } from './AlbumList.style';
import { Album } from '../Album/Album';

export const AlbumList = (prop) => {
  const { showList } = prop;

  const renderedListItems = showList.map((album) => {
    const id = album.id.attributes['im:id'];
    return <Album key={id} data-id={id} album={album} />;
  });

  return <AlbumListWrapper>{renderedListItems}</AlbumListWrapper>;
};
