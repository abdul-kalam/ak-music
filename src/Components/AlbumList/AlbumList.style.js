import styled from 'styled-components';
export const AlbumListWrapper = styled.ul`
display: flex;
flex-wrap:wrap;
@media only screen and (max-width: 768px) {
    justify-content: center;
  }
`;