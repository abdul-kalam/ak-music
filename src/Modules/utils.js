export const getLocalStorageValue = (key) => {
  const value = localStorage.getItem(key);
  if (value) {
    return JSON.parse(value);
  }
  return '';
};

export const setLocalStorageValue = (key, value) => {
  value = JSON.stringify(value);
  localStorage.setItem(key, value);
};
