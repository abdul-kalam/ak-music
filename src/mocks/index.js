export const mockTopAlbum = {
  feed: {
    author: {
      name: {
        label: 'iTunes Store',
      },
      uri: {
        label: 'http://www.apple.com/itunes/',
      },
    },
    entry: [
      {
        isFavorite: true,
        'im:name': {
          label: 'Welcome 2 America',
        },
        'im:image': [
          {
            label:
              'https://is4-ssl.mzstatic.com/image/thumb/Music114/v4/55/46/19/55461993-6fd7-d703-9b39-03043f585203/886449187249.jpg/55x55bb.png',
            attributes: {
              height: '55',
            },
          },
          {
            label:
              'https://is3-ssl.mzstatic.com/image/thumb/Music114/v4/55/46/19/55461993-6fd7-d703-9b39-03043f585203/886449187249.jpg/60x60bb.png',
            attributes: {
              height: '60',
            },
          },
          {
            label:
              'https://is1-ssl.mzstatic.com/image/thumb/Music114/v4/55/46/19/55461993-6fd7-d703-9b39-03043f585203/886449187249.jpg/170x170bb.png',
            attributes: {
              height: '170',
            },
          },
        ],
        'im:itemCount': {
          label: '12',
        },
        'im:price': {
          label: '$9.99',
          attributes: {
            amount: '9.99',
            currency: 'USD',
          },
        },
        'im:contentType': {
          'im:contentType': {
            attributes: {
              term: 'Album',
              label: 'Album',
            },
          },
          attributes: {
            term: 'Music',
            label: 'Music',
          },
        },
        rights: {
          label:
            '℗ 2021 NPG Records, Inc. under exclusive license to Legacy Recordings',
        },
        title: {
          label: 'Welcome 2 America - Prince',
        },
        link: {
          attributes: {
            rel: 'alternate',
            type: 'text/html',
            href: 'https://music.apple.com/us/album/welcome-2-america/1561643530?uo=2',
          },
        },
        id: {
          label:
            'https://music.apple.com/us/album/welcome-2-america/1561643530?uo=2',
          attributes: {
            'im:id': '1561643530',
          },
        },
        'im:artist': {
          label: 'Prince',
          attributes: {
            href: 'https://music.apple.com/us/artist/prince/155814?uo=2',
          },
        },
        category: {
          attributes: {
            'im:id': '15',
            term: 'R&B/Soul',
            scheme: 'https://music.apple.com/us/genre/music-r-b-soul/id15?uo=2',
            label: 'R&B/Soul',
          },
        },
        'im:releaseDate': {
          label: '2021-07-30T00:00:00-07:00',
          attributes: {
            label: 'July 30, 2021',
          },
        },
      },
      {
        isFavorite: true,
        'im:name': {
          label: 'Into The Mystery',
        },
        'im:image': [
          {
            label:
              'https://is2-ssl.mzstatic.com/image/thumb/Music125/v4/f5/48/5f/f5485f4c-458e-425e-fa60-e3ce2871cb00/075679788856.jpg/55x55bb.png',
            attributes: {
              height: '55',
            },
          },
          {
            label:
              'https://is4-ssl.mzstatic.com/image/thumb/Music125/v4/f5/48/5f/f5485f4c-458e-425e-fa60-e3ce2871cb00/075679788856.jpg/60x60bb.png',
            attributes: {
              height: '60',
            },
          },
          {
            label:
              'https://is2-ssl.mzstatic.com/image/thumb/Music125/v4/f5/48/5f/f5485f4c-458e-425e-fa60-e3ce2871cb00/075679788856.jpg/170x170bb.png',
            attributes: {
              height: '170',
            },
          },
        ],
        'im:itemCount': {
          label: '12',
        },
        'im:price': {
          label: '$7.99',
          attributes: {
            amount: '7.99',
            currency: 'USD',
          },
        },
        'im:contentType': {
          'im:contentType': {
            attributes: {
              term: 'Album',
              label: 'Album',
            },
          },
          attributes: {
            term: 'Music',
            label: 'Music',
          },
        },
        rights: {
          label: '℗ 2021 Elektra Records LLC',
        },
        title: {
          label: 'Into The Mystery - NEEDTOBREATHE',
        },
        link: {
          attributes: {
            rel: 'alternate',
            type: 'text/html',
            href: 'https://music.apple.com/us/album/into-the-mystery/1566670096?uo=2',
          },
        },
        id: {
          label:
            'https://music.apple.com/us/album/into-the-mystery/1566670096?uo=2',
          attributes: {
            'im:id': '1566670096',
          },
        },
        'im:artist': {
          label: 'NEEDTOBREATHE',
          attributes: {
            href: 'https://music.apple.com/us/artist/needtobreathe/113247214?uo=2',
          },
        },
        category: {
          attributes: {
            'im:id': '21',
            term: 'Rock',
            scheme: 'https://music.apple.com/us/genre/music-rock/id21?uo=2',
            label: 'Rock',
          },
        },
        'im:releaseDate': {
          label: '2021-07-30T00:00:00-07:00',
          attributes: {
            label: 'July 30, 2021',
          },
        },
      },
      {
        'im:name': {
          label: 'The House Is Burning',
        },
        'im:image': [
          {
            label:
              'https://is2-ssl.mzstatic.com/image/thumb/Music115/v4/3f/28/c8/3f28c8e9-c28a-1087-8ccb-654c35bdfbb8/093624878070.jpg/55x55bb.png',
            attributes: {
              height: '55',
            },
          },
          {
            label:
              'https://is4-ssl.mzstatic.com/image/thumb/Music115/v4/3f/28/c8/3f28c8e9-c28a-1087-8ccb-654c35bdfbb8/093624878070.jpg/60x60bb.png',
            attributes: {
              height: '60',
            },
          },
          {
            label:
              'https://is3-ssl.mzstatic.com/image/thumb/Music115/v4/3f/28/c8/3f28c8e9-c28a-1087-8ccb-654c35bdfbb8/093624878070.jpg/170x170bb.png',
            attributes: {
              height: '170',
            },
          },
        ],
        'im:itemCount': {
          label: '16',
        },
        'im:price': {
          label: '$9.99',
          attributes: {
            amount: '9.99',
            currency: 'USD',
          },
        },
        'im:contentType': {
          'im:contentType': {
            attributes: {
              term: 'Album',
              label: 'Album',
            },
          },
          attributes: {
            term: 'Music',
            label: 'Music',
          },
        },
        rights: {
          label:
            '℗ 2021 Top Dawg Entertainment, under exclusive license to Warner Records Inc.',
        },
        title: {
          label: 'The House Is Burning - Isaiah Rashad',
        },
        link: {
          attributes: {
            rel: 'alternate',
            type: 'text/html',
            href: 'https://music.apple.com/us/album/the-house-is-burning/1577158260?uo=2',
          },
        },
        id: {
          label:
            'https://music.apple.com/us/album/the-house-is-burning/1577158260?uo=2',
          attributes: {
            'im:id': '1577158260',
          },
        },
        'im:artist': {
          label: 'Isaiah Rashad',
          attributes: {
            href: 'https://music.apple.com/us/artist/isaiah-rashad/605391263?uo=2',
          },
        },
        category: {
          attributes: {
            'im:id': '18',
            term: 'Hip Hop/Rap',
            scheme:
              'https://music.apple.com/us/genre/music-hip-hop-rap/id18?uo=2',
            label: 'Hip-Hop/Rap',
          },
        },
        'im:releaseDate': {
          label: '2021-07-30T00:00:00-07:00',
          attributes: {
            label: 'July 30, 2021',
          },
        },
      },
    ],
    updated: {
      label: '2021-08-04T13:45:32-07:00',
    },
    rights: {
      label: 'Copyright 2008 Apple Inc.',
    },
    title: {
      label: 'iTunes Store: Top Albums',
    },
    icon: {
      label: 'http://itunes.apple.com/favicon.ico',
    },
    link: [
      {
        attributes: {
          rel: 'alternate',
          type: 'text/html',
          href: 'https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewTop?cc=us&id=1&popId=11',
        },
      },
      {
        attributes: {
          rel: 'self',
          href: 'https://mzstoreservices-int-st.itunes.apple.com/us/rss/topalbums/limit=100/json',
        },
      },
    ],
    id: {
      label:
        'https://mzstoreservices-int-st.itunes.apple.com/us/rss/topalbums/limit=100/json',
    },
  },
  titleListMock: [
    {
      id: '1566670096',
      name: 'Welcome to America',
      image: 'www.test.com/img.jpg',
    },
    {
      id: '1566670094',
      name: 'Welcome to India',
      image: 'www.test.com/img1.jpg',
    },
  ],
};

export const mockAlbumSongs = {
  resultCount: 4,
  results: [
    {
      wrapperType: 'collection',
      collectionType: 'Album',
      artistId: 4779205,
      collectionId: 1570554138,
      amgArtistId: 825908,
      artistName: 'Chris Young',
      collectionName: 'Famous Friends',
      collectionCensoredName: 'Famous Friends',
      artistViewUrl:
        'https://music.apple.com/us/artist/chris-young/4779205?uo=4',
      collectionViewUrl:
        'https://music.apple.com/us/album/famous-friends/1570554138?uo=4',
      artworkUrl60:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/60x60bb.jpg',
      artworkUrl100:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/100x100bb.jpg',
      collectionPrice: 11.99,
      collectionExplicitness: 'notExplicit',
      trackCount: 14,
      copyright: '℗ 2021 Sony Music Entertainment. All rights reserved.',
      country: 'USA',
      currency: 'USD',
      releaseDate: '2021-08-06T07:00:00Z',
      primaryGenreName: 'Country',
    },
    {
      wrapperType: 'track',
      kind: 'song',
      artistId: 4779205,
      collectionId: 1570554138,
      trackId: 1570554142,
      artistName: 'Chris Young',
      collectionName: 'Famous Friends',
      trackName: 'Raised on Country',
      collectionCensoredName: 'Famous Friends',
      trackCensoredName: 'Raised on Country',
      artistViewUrl:
        'https://music.apple.com/us/artist/chris-young/4779205?uo=4',
      collectionViewUrl:
        'https://music.apple.com/us/album/raised-on-country/1570554138?i=1570554142&uo=4',
      trackViewUrl:
        'https://music.apple.com/us/album/raised-on-country/1570554138?i=1570554142&uo=4',
      previewUrl:
        'https://audio-ssl.itunes.apple.com/itunes-assets/AudioPreview115/v4/10/c0/3f/10c03f67-bb34-ff6b-403b-4c6070fb56ad/mzaf_15785209836383516861.plus.aac.p.m4a',
      artworkUrl30:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/30x30bb.jpg',
      artworkUrl60:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/60x60bb.jpg',
      artworkUrl100:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/100x100bb.jpg',
      collectionPrice: 11.99,
      trackPrice: 1.29,
      releaseDate: '2019-01-04T08:00:00Z',
      collectionExplicitness: 'notExplicit',
      trackExplicitness: 'notExplicit',
      discCount: 1,
      discNumber: 1,
      trackCount: 14,
      trackNumber: 1,
      trackTimeMillis: 176520,
      country: 'USA',
      currency: 'USD',
      primaryGenreName: 'Country',
      isStreamable: true,
    },
    {
      wrapperType: 'track',
      kind: 'song',
      artistId: 4779205,
      collectionId: 1570554138,
      trackId: 1570554143,
      artistName: 'Chris Young & Kane Brown',
      collectionName: 'Famous Friends',
      trackName: 'Famous Friends',
      collectionCensoredName: 'Famous Friends',
      trackCensoredName: 'Famous Friends',
      collectionArtistName: 'Chris Young',
      artistViewUrl:
        'https://music.apple.com/us/artist/chris-young/4779205?uo=4',
      collectionViewUrl:
        'https://music.apple.com/us/album/famous-friends/1570554138?i=1570554143&uo=4',
      trackViewUrl:
        'https://music.apple.com/us/album/famous-friends/1570554138?i=1570554143&uo=4',
      previewUrl:
        'https://audio-ssl.itunes.apple.com/itunes-assets/AudioPreview125/v4/42/7a/5f/427a5f4b-5e18-fdb6-3ba8-56a2d06483b0/mzaf_2809262009335345042.plus.aac.p.m4a',
      artworkUrl30:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/30x30bb.jpg',
      artworkUrl60:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/60x60bb.jpg',
      artworkUrl100:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/100x100bb.jpg',
      collectionPrice: 11.99,
      trackPrice: 1.29,
      releaseDate: '2020-11-20T08:00:00Z',
      collectionExplicitness: 'notExplicit',
      trackExplicitness: 'notExplicit',
      discCount: 1,
      discNumber: 1,
      trackCount: 14,
      trackNumber: 2,
      trackTimeMillis: 166507,
      country: 'USA',
      currency: 'USD',
      primaryGenreName: 'Country',
      isStreamable: true,
    },
    {
      wrapperType: 'track',
      kind: 'song',
      artistId: 4779205,
      collectionId: 1570554138,
      trackId: 1570554144,
      artistName: 'Chris Young & Lauren Alaina',
      collectionName: 'Famous Friends',
      trackName: "Town Ain't Big Enough",
      collectionCensoredName: 'Famous Friends',
      trackCensoredName: "Town Ain't Big Enough",
      collectionArtistName: 'Chris Young',
      artistViewUrl:
        'https://music.apple.com/us/artist/chris-young/4779205?uo=4',
      collectionViewUrl:
        'https://music.apple.com/us/album/town-aint-big-enough/1570554138?i=1570554144&uo=4',
      trackViewUrl:
        'https://music.apple.com/us/album/town-aint-big-enough/1570554138?i=1570554144&uo=4',
      previewUrl:
        'https://audio-ssl.itunes.apple.com/itunes-assets/AudioPreview115/v4/d3/0d/69/d30d693f-570d-0bc0-3c70-7c7a272baec3/mzaf_7645675846915846385.plus.aac.p.m4a',
      artworkUrl30:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/30x30bb.jpg',
      artworkUrl60:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/60x60bb.jpg',
      artworkUrl100:
        'https://is1-ssl.mzstatic.com/image/thumb/Music115/v4/d1/6e/a9/d16ea9a6-c2a4-90aa-99e7-a981ee8cb689/source/100x100bb.jpg',
      collectionPrice: 11.99,
      trackPrice: 1.29,
      releaseDate: '2019-07-12T07:00:00Z',
      collectionExplicitness: 'notExplicit',
      trackExplicitness: 'notExplicit',
      discCount: 1,
      discNumber: 1,
      trackCount: 14,
      trackNumber: 3,
      trackTimeMillis: 171800,
      country: 'USA',
      currency: 'USD',
      primaryGenreName: 'Country',
      isStreamable: true,
    },
  ],
};
