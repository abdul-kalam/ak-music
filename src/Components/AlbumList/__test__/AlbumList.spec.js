import React from 'react';
import { render, screen } from '../../../Modules/test-utils';
import { AlbumList } from './../AlbumList';
import { mockTopAlbum } from './../../../mocks';

describe('Album List component', () => {
  test('component should render list of albums correctly', () => {
    const mockAlbum = mockTopAlbum.feed.entry;

    const { asFragment } = render(<AlbumList showList={mockAlbum} />);

    expect(screen.getAllByAltText('Welcome 2 America')).toHaveLength(1);
    expect(screen.getByAltText('Into The Mystery')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
