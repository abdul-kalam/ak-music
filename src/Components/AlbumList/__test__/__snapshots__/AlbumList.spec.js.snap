// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album List component component should render list of albums correctly 1`] = `
<DocumentFragment>
  <ul
    class="sc-bdnxRM cHtmfc"
  >
    <li
      class="sc-gtsrHT iKIoUa"
      data-testid="album"
    >
      <div
        class="sc-dlnjwi eEhPkN"
      >
        <div
          class="sc-hKFxyN hFgHbL"
        >
          <img
            alt="Welcome 2 America"
            class="sc-fujyAs dMRvub"
            loading="lazy"
            src="https://is1-ssl.mzstatic.com/image/thumb/Music114/v4/55/46/19/55461993-6fd7-d703-9b39-03043f585203/886449187249.jpg/170x170bb.png"
          />
        </div>
        <div
          class="sc-eCApnc hrSlJz"
        >
          <div
            class="sc-jSFjdj kUXCdg"
          >
            <span
              class="sc-gKAaRy kkQDiB"
            >
              Welcome 2 America
            </span>
            <div
              class="sc-iCoGMd itDCGY"
            >
              Prince
            </div>
          </div>
          <div
            class="sc-pNWdM eOVURI"
          >
            <svg
              height="30px"
              id="Layer_1"
              version="1.1"
              viewBox="0 0 455 455"
              width="30px"
              x="0px"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0px"
            >
              <title>
                Favorite
              </title>
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="2"
              >
                <g
                  fill="#fa2d48"
                  fill-rule="nonzero"
                >
                  <path
                    d="M326.632,10.346c-38.733,0-74.991,17.537-99.132,46.92c-24.141-29.383-60.399-46.92-99.132-46.92 C57.586,10.346,0,67.931,0,138.714c0,55.426,33.049,119.535,98.23,190.546c50.162,54.649,104.729,96.96,120.257,108.626l9.01,6.769 l9.009-6.768c15.53-11.667,70.099-53.979,120.26-108.625C421.95,258.251,455,194.141,455,138.714 C455,67.931,397.414,10.346,326.632,10.346z"
                  />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
    </li>
    <li
      class="sc-gtsrHT iKIoUa"
      data-testid="album"
    >
      <div
        class="sc-dlnjwi eEhPkN"
      >
        <div
          class="sc-hKFxyN hFgHbL"
        >
          <img
            alt="Into The Mystery"
            class="sc-fujyAs dMRvub"
            loading="lazy"
            src="https://is2-ssl.mzstatic.com/image/thumb/Music125/v4/f5/48/5f/f5485f4c-458e-425e-fa60-e3ce2871cb00/075679788856.jpg/170x170bb.png"
          />
        </div>
        <div
          class="sc-eCApnc hrSlJz"
        >
          <div
            class="sc-jSFjdj kUXCdg"
          >
            <span
              class="sc-gKAaRy kkQDiB"
            >
              Into The Mystery
            </span>
            <div
              class="sc-iCoGMd itDCGY"
            >
              NEEDTOBREATHE
            </div>
          </div>
          <div
            class="sc-pNWdM eOVURI"
          >
            <svg
              height="30px"
              id="Layer_1"
              version="1.1"
              viewBox="0 0 455 455"
              width="30px"
              x="0px"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0px"
            >
              <title>
                Favorite
              </title>
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="2"
              >
                <g
                  fill="#fa2d48"
                  fill-rule="nonzero"
                >
                  <path
                    d="M326.632,10.346c-38.733,0-74.991,17.537-99.132,46.92c-24.141-29.383-60.399-46.92-99.132-46.92 C57.586,10.346,0,67.931,0,138.714c0,55.426,33.049,119.535,98.23,190.546c50.162,54.649,104.729,96.96,120.257,108.626l9.01,6.769 l9.009-6.768c15.53-11.667,70.099-53.979,120.26-108.625C421.95,258.251,455,194.141,455,138.714 C455,67.931,397.414,10.346,326.632,10.346z"
                  />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
    </li>
    <li
      class="sc-gtsrHT iKIoUa"
      data-testid="album"
    >
      <div
        class="sc-dlnjwi eEhPkN"
      >
        <div
          class="sc-hKFxyN hFgHbL"
        >
          <img
            alt="The House Is Burning"
            class="sc-fujyAs dMRvub"
            loading="lazy"
            src="https://is3-ssl.mzstatic.com/image/thumb/Music115/v4/3f/28/c8/3f28c8e9-c28a-1087-8ccb-654c35bdfbb8/093624878070.jpg/170x170bb.png"
          />
        </div>
        <div
          class="sc-eCApnc hrSlJz"
        >
          <div
            class="sc-jSFjdj kUXCdg"
          >
            <span
              class="sc-gKAaRy kkQDiB"
            >
              The House Is Burning
            </span>
            <div
              class="sc-iCoGMd itDCGY"
            >
              Isaiah Rashad
            </div>
          </div>
          <div
            class="sc-pNWdM eOVURI"
          >
            <svg
              height="30px"
              id="Layer_1"
              version="1.1"
              viewBox="0 0 455 455"
              width="30px"
              x="0px"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0px"
            >
              <title>
                UnFavorite
              </title>
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="none"
                stroke-width="2"
              >
                <g
                  fill="#fa2d48"
                  fill-rule="nonzero"
                >
                  <path
                    d="M326.632,10.346c-38.733,0-74.991,17.537-99.132,46.92c-24.141-29.384-60.398-46.92-99.132-46.92 C57.586,10.346,0,67.931,0,138.714c0,55.426,33.05,119.535,98.23,190.546c50.161,54.647,104.728,96.959,120.257,108.626l9.01,6.769 l9.01-6.768c15.529-11.667,70.098-53.978,120.26-108.625C421.949,258.251,455,194.141,455,138.714 C455,67.931,397.414,10.346,326.632,10.346z M334.666,308.974c-41.259,44.948-85.648,81.283-107.169,98.029 c-21.52-16.746-65.907-53.082-107.166-98.03C61.236,244.592,30,185.717,30,138.714c0-54.24,44.128-98.368,98.368-98.368 c35.694,0,68.652,19.454,86.013,50.771l13.119,23.666l13.119-23.666c17.36-31.316,50.318-50.771,86.013-50.771 c54.24,0,98.368,44.127,98.368,98.368C425,185.719,393.763,244.594,334.666,308.974z"
                  />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
    </li>
  </ul>
</DocumentFragment>
`;
